api/graphql/schema.graphql

```diff
@@ -1675,10 +1683,26 @@ input UpdateScreeningQuestionResponsesHasMany {
   update: [UpdateScreeningQuestionResponseInput!]
   delete: [ID!]
 }
+
+input UpdateEducationExperiencesRequirementBelongsToMany {
+  connect: [ID!]
+  sync: [ID!]
+  syncWithoutDetaching: [ID!]
+  disconnect: [ID!]
```

Comment:
So far, we've tended to just define `sync` unless we have a good reason to add more options. In this case, I think we can simplify by just having `sync`.